hmrc-mongo-play-26/src/main/scala/uk/gov/hmrc/mongo/play/json/MongoJavatimeFormats.scala

```diff
@@ -80,9 +65,9 @@ trait MongoJavatimeFormats {
   val localDateTimeFormats = Format(localDateTimeRead, localDateTimeWrite)
 
   trait Implicits {
-    implicit val instantFormats: Format[Instant]             = outer.instantFormats
-    implicit val localDateFormats: Format[LocalDate]         = outer.localDateFormats
-    implicit val localDateTimeFormats: Format[LocalDateTime] = outer.localDateTimeFormats
+    implicit val jatInstantFormats: Format[Instant]             = outer.instantFormats
```

Comment:
what's `jat`?

Reply:
implicits need unique names - the name is unimportant (non-implicit usage should be using the MongoJavatimeFormats.instantFormats).
I've just gone with jot for jodatime and jat for javatime